modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java

```diff
@@ -1115,6 +1131,18 @@ public CommerceOrder reorderCommerceOrder(
 				serviceContext);
 		}
 
+		if (recalculate) {
+			newCommerceOrder =
+				commerceOrderLocalService.updateCommerceShippingMethod(
```

Comment:
why do you need to update the shipping method here?

Reply:
@luca-pellizzon , we need the logic inside this if check here to run so we are able to calculate the correct shipping amount:
https://github.com/liferay-commerce/liferay-portal/blob/ef568a7db2d675c9865d5e268980aa2b3cb1a2c1/modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java#L1979

Otherwise, if you make a USD order, and you click reorder while the new currency is Chinese Yuan (Example we used in the ticket), the shipping amount will incorrectly still show as $15.00 instead of the converted value in Chinese Yuan.


Sincerely,
Brian I. Kim